internal/controller/server/repository/metadata/metadata.go

```diff
@@ -1,48 +1,56 @@
-package repository
+package metadata
 
 import (
 	"context"
 	"errors"
+	"fmt"
+	"octavius/internal/pkg/log"
+	"octavius/internal/pkg/util"
+
+	"google.golang.org/grpc/status"
+
```

Comment:
are these imports sorted?

Reply:
looks sorted, but let's re-run go fmt to make sure.